pkg/container/docker_cli.go

```diff
@@ -0,0 +1,1078 @@
+//nolint:unparam,errcheck,depguard,deadcode,unused
+package container
+
+// exact copy of https://github.com/docker/cli/blob/9ac8584acfd501c3f4da0e845e3a40ed15c85041/cli/command/container/opts.go
+
```

Comment:
Since this is a copy, we have to add the license to this file.
We are not allowed to copy their code as is without keeping the license.

Reply:
Add header to this file and DOCKER_LICENSE with full text.
Please check